addons/mail_group/models/mail_group.py

```diff
@@ -355,7 +350,7 @@ def message_post(self, body='', subject=None, email_from=None, author_id=None, *
         elif moderation_rule and moderation_rule.status == 'ban':
             group_message.action_moderate_reject()
 
-        elif self.moderation_notify:
+        elif self.moderation_notify_msg:
```

Comment:
Would probably also check moderation_notify like before, in addition to message (in case someone unchecked moderation_notify but kept message)